packages/amplication-build-manager/src/code-generator/code-generator-splitter.service.ts

```diff
@@ -1,17 +1,34 @@
 import { DSGResourceData } from "@amplication/code-gen-types";
 import { Injectable } from "@nestjs/common";
 import { cloneDeep } from "lodash";
+import {
+  BuildId,
+  JobBuildId,
+  EnumDomainName,
+  EnumJobStatus,
+  RedisValue,
+} from "../types";
+import { RedisService } from "../redis/redis.service";
+import { AmplicationLogger } from "@amplication/util/nestjs/logging";
 
-export enum EnumDomainType {
-  Server = "server",
-  AdminUI = "admin-ui",
-}
-
-type ResourceTuple = [EnumDomainType, DSGResourceData];
-
+type ResourceTuple = [JobBuildId<BuildId>, DSGResourceData];
 @Injectable()
 export class CodeGeneratorSplitterService {
-  splitJobs(dsgResourceData: DSGResourceData): ResourceTuple[] {
+  // TODO: rename to CodeGeneratorJobHandlerService (?)
```

Comment:
Let's avoid adding redundant comments

Reply:
This is for me. I will remove it after we merge it to the base branch and rename the class name